docs/Change_Log.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## API v9
+
+#### April 28, 2021
+
+API v9 is now available.
+
+API v9 includes supports for [threads](#DOCS_TOPICS_THREADS), an upcoming feature.  Older API versions will not receive any Gateway Events for threads, so it is important to update soon!  We've prepared a [migration guide](#DOCS_TOPICS_THREADS) to help make the upgrade process very straightforward.
```

Comment:
not entirely sure where this info goes but we also removed the deprecated `/channels/:id/messages/:id/suppress-embeds` route in v9.